elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/EntityProjectionTranslator.java

```diff
@@ -46,13 +49,20 @@
     private FilterExpression whereFilter;
     private FilterExpression havingFilter;
     private EntityDictionary dictionary;
+    private Boolean bypassCache;
+
 
     public EntityProjectionTranslator(QueryEngine engine, Table table,
-                                      EntityProjection entityProjection, EntityDictionary dictionary) {
+                                      EntityProjection entityProjection, EntityDictionary dictionary,
+                                      Optional<MultivaluedMap<String, String>> requestHeaders) {
         this.engine = engine;
         this.queriedTable = table;
         this.entityProjection = entityProjection;
         this.dictionary = dictionary;
+        String bypassCache = (requestHeaders.isPresent() && requestHeaders.get().get("bypasscache") != null)
+                             ? requestHeaders.get().get("bypasscache").get(0) : null;
+        String safebypassCache = bypassCache == null ? "true" : bypassCache;
```

Comment:
If bypassCache is null, shouldn't this be "false" instead of "true"?  Also, rather than parse the boolean, I would look for "true" and set it or otherwise it should always just be false (regardless of the value).